src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/TextFieldElementDriver.cs

```diff
@@ -79,7 +80,7 @@ public class TextFieldElementDriver : FormsElementDriver<TextField>{
         protected override void OnDisplaying(TextField element, ElementDisplayContext context) {
             context.ElementShape.ProcessedName = _tokenizer.Replace(element.Name, context.GetTokenData());
             context.ElementShape.ProcessedLabel = _tokenizer.Replace(element.Label, context.GetTokenData());
-            context.ElementShape.ProcessedValue = _tokenizer.Replace(element.RuntimeValue, context.GetTokenData());
+            context.ElementShape.ProcessedValue = HttpUtility.HtmlDecode(_tokenizer.Replace(element.RuntimeValue, context.GetTokenData()));
```

Comment:
@sebastienros You were right regarding the security issue. If you put at least two fields on a form, one of which being required and the other one providing a token, that token gets evaluated and displayed (due to the model validation error the form gets displayed and submitted values are shown). Fixing this.